ood-portal-generator/spec/fixtures/ood-portal.conf.default.deb

```diff
@@ -1,150 +0,0 @@
-#
```

Comment:
Might be good to replace this fixture with a test that tests the behavior for Debian based systems when `auth` is defined.  I think it was a good test to validate the difference between RHEL and Debian based systems.  I think the only difference by default is going to be the log path.

Reply:
will do

Reply:
Fixed this in 30eea5e7770d4fa985aeb0fdcfaacb66c50e2fa4